explainer.md

```diff
@@ -33,11 +33,11 @@ First, we need a mechanism for attaching additional information to an event list
   document.addEventListener('touchstart', handler, {capture: true});
 ```
 
-This much is simply new (extensible) syntax for existing behavior - specifying [whether you want the listener invoked during the capture phase or bubbling phase](http://javascript.info/tutorial/bubbling-and-capturing#capturing).
+This is simply the new (extensible) syntax for existing behavior - specifying [whether you want the listener invoked during the capture phase or bubbling phase](http://javascript.info/tutorial/bubbling-and-capturing#capturing).
 
 ## Feature Detection
 
-Because older browsers will interpret any object in the 3rd argument as `useCapture=true` it's important for developers to use feature detection or [a polyfill](https://github.com/WICG/EventListenerOptions/blob/gh-pages/EventListenerOptions.polyfill.js) when using this API.  Feature detection for specific options can be done as follows:
+Because older browsers will interpret any object in the 3rd argument as `capture=true`, it's important for developers to use feature detection or [a polyfill](https://github.com/WICG/EventListenerOptions/blob/gh-pages/EventListenerOptions.polyfill.js) when using this API, to avoid unforeseen results.  Feature detection for specific options can be done as follows:
```

Comment:
I was using `useCapture` because that was the old name of the [addEventListener argument](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener).  This is distinct from (but same semantics as) the `capture` member of `EventListenerOptions`.  This issue is pedantic though so probably not of interest to most web developers.  Could also say "as a true value for the capture argument".  I don't know, whatever you think is clearer is fine with me, just wanted to explain the distinction.